FlowCrypt/Models/Realm Models/FolderRealmObject.swift

```diff
@@ -10,23 +10,21 @@ import Foundation
 import RealmSwift
 
 final class FolderRealmObject: Object {
-    @Persisted(primaryKey: true) var path: String = ""
```

Comment:
@sosnovsky
Same for the default values for Realm properties. Why we need them?

Reply:
Absolutely no need for default values for Realm properties, except when we specifically want them. 

As it was before, it was wrong.

Reply:
Agree, no need for default values here